streams/examples/src/main/java/org/apache/kafka/streams/examples/pageview/PageViewTypedDemo.java

```diff
@@ -160,7 +167,8 @@ public PageViewByRegion apply(PageView view, UserProfile profile) {
                         return new KeyValue<>(viewRegion.region, viewRegion);
                     }
                 })
-                .countByKey(TimeWindows.of("GeoPageViewsWindow", 7 * 24 * 60 * 60 * 1000L).advanceBy(1000), Serdes.String())
+                .groupByKey(Serdes.String(), pageViewByRegionSerde)
```

Comment:
Why do we need to provide a serde here, as it should not need to be materialized / repartitioned before aggregation?